### DISCUSSION TITLE
adminsortable2 - TemplateNotFound

### DISCUSSION THREAD

**Discussion by belthaZornv** [jrief/django-admin-sortable2#361] [Announcements]
After installing the library, and adding to the apps and all the necessities; I've been faced with this error:

```
TemplateDoesNotExist at /platform/flow/flow/add/
adminsortable2/edit_inline/tabular-django-3.2.html
```

That seems to come from this:

```
class SortableTabularInline(SortableInlineAdminMixin, admin.TabularInline):
    template = 'adminsortable2/edit_inline/tabular-django-{0}.{1}.html'.format(*DJANGO_VERSION)
```

Is there something I'm missing?

My code:

```
class FlowActionInline(SortableTabularInline, admin.TabularInline):
    fields = ('action',)

    model = FlowAction

```

**Comment by marksweb** [comment]
Seems valid. I can see one template, but the code you've referenced is loading different templates.

**Comment by jrief** [comment]
Apparently you attempted to use **django-admin-sortable2** with Django-3.2.

Version 2 only works with [Django-4.0 and later](https://github.com/jrief/django-admin-sortable2/blob/6fa6c67245656eb205634b7d8b11bac191d482b6/setup.py#L48).
